cypress/support/step-definitions/common-steps.js

```diff
@@ -90,6 +90,11 @@ Given('I open {string} component in iframe', (component) => {
   visitComponentUrl(component, 'default', true);
 });
 
+// the step above should be refactored and changed to in noiFrame
+Given('I open {string} component in noiFrame', (component) => {
```

Comment:
if you can use this step don't creat new one
`Given('I open basic Test {string} component page in noIframe', (component) => {
  visitComponentUrl(component, 'basic', true, 'test-');
});`

Reply:
in step you are mentioned we are using `basic` suffix, but Action Popover has `default` one